README.md

```diff
@@ -1,2 +1,82 @@
 # PaymentSDK
 SDK for payment backends (marketplaces etc)
+
+#### Motivation : 
+
+Using this SDK sellers will be able to process Gooddollar payments through our API.
+
+For example:
+A Seller like e-commerce for "brand" would like to be able to request payment from his customers at Gooddollar.
+
+### Details
+#### Seller API
+A typescript NPM module that works both on browser/node that implements the following methods
+
+##### Method 1
+```
+generatePaymentLink(PaymentDetails:string)
+```
+
+Creates a payment link.
+Example:
+[http://wallet.gooddollar.org:3000/? code=eyJtIjoiM2RQQjRxUjNBZUoyY0tzalJnYWp3dE1hUms3TjI0aFVMZzYiLCJhIjoiM...]()
+the query param code is the base64 encrypted value returned from generateCode.
+
+The generated code will return an object with the paymenDetails params and the ones that we already have
+
+Example:
```

Comment:
this is not an example this is an explanation of how the input is encoded in the base64